resources/tentative/newssite/news-nuxt/app.vue

```diff
@@ -0,0 +1,34 @@
+<script setup>
+useHead({
+    title: "The Daily Broadcast",
+    htmlAttrs: {
+        lang: "en",
+    },
+    meta: [
+        { charset: "utf-8" },
+        { name: "viewport", content: "width=device-width, initial-scale=1" },
+        {
+            hid: "description",
+            name: "description",
+            content: "A news site developed with Nuxt.",
+        },
+    ],
+});
+</script>
+
+<script>
+  // "Fixes" an issue with calling history.replaceState too often in certain browsers during testing.
+  // Note: This should NOT be used in a production application.
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  history.replaceState = function(state) {
```

Comment:
@rniwa  - the security error popped up again when adding the test initially. 
Overriding history.replaceState in the workload itself results into not throwing the error anymore. 
The app still functions as it should and it also doesn't have any impact on scores. 

Could you test on your side if that works?